Assets/VRTK/SDK/Daydream/SDK_DaydreamController.cs

```diff
@@ -0,0 +1,266 @@
+﻿// Daydream Controller|SDK_Daydream|003
+namespace VRTK
+{
+#if VRTK_SDK_DAYDREAM
+    using UnityEngine;
+    using System.Collections.Generic;
+
+    public class SDK_DaydreamController : SDK_FallbackController
+    {
+        private GameObject controller;
+
+        private Vector3 prevPosition;
+        private Vector3 prevRotation;
+        private Vector3 velocity = Vector3.zero;
+        private Vector3 angularVelocity = Vector3.zero;
+
+        //TODO: where should we check that controller is connected? eg if (GvrController.State != GvrConnectionState.Connected) 
```

Comment:
Are you getting a Debug.Log error then? If you are then I'd recommend removing the `Awake` here and I'll fix it in the Fallback because it shouldn't be bubbling the error through to inherited classes, otherwise you have to patch it with empty `Awake()` methods ;) :p